### DISCUSSION TITLE
Track gitops related future events

### DISCUSSION THREAD

**Discussion by samueltauil** [open-gitops/project#165] [Ideas]
It's been a common and recurrent discussion around future events, how and when we can interact, so my suggestion would be creating a live document where we can have visibility in upcoming events/locations/dates so we can plan better maybe.

Thoughts?

**Comment by christianh814** [comment]
What kind of document?

**Reply by samueltauil** [reply]
Perhaps a google docs sheet, @JaimeMagiera any thoughts?
